interview-questions.md

```diff
@@ -14,44 +14,44 @@ Researched answer:
 
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer: Git is the software tool to store code data. Github is a tool to share code data.
 
-Researched answer:
+Researched answer: Devops tool for source code management allows multiple developers to work together on projects. GitHub is a code hosting platform allows you to work with others on project from anywhere in the world.
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: conditional operators,
 
-Researched answer:
+Researched answer: &&, || ! logical operators
 
 3. What is an index? What is the difference between index and value?
 
-Your answer:
+Your answer: Index starts the count from 0 of items in a variable. Value is the actual number or item itself, what it represents.
 
-Researched answer:
+Researched answer: Index contains the position of a regular expression match within a string. A valuecan bea number, text, null, boolean or string, they are given a definition and recognized by javascript.
 
 4. Compare and contrast arrays and strings. Please include ways in which they are the same and ways in which they are different.
 
-Your answer:
+Your answer: Array's are a collection of items within a bracket, they can be numbers, or even include strings. A string is a collection of characters that will read as text. numbers can also be in a string but a string has quotations.
 
-Researched answer:
+Researched answer: A string is a collection of characters while an Array is a data structure that holds a collection of elements.
```

Comment:
You should expand a bit more on your researched answers. How are they the same? Do they both have similar built-in methods?